docs/pages/toolchain/katana/genesis.md

```diff
@@ -149,11 +149,11 @@ The genesis config file is a JSON file that contains the following fields:
 	"classes": [
 		{
 			"class": "path/to/file/erc20.json",
-			"classHash": "0x8"
+			"name": "MyClass"
 		},
 		{
 			"class": "path/to/file/universal_deployer.json",
-			"classHash": "0x444"
+			"classHash": "AnotherClass"
```

Comment:
class hash should be a 0x-prefixed hex value and not doesnt accept arbitrary string. that would be for the `name` field